aiblog-be/controllers/postController.js

```diff
@@ -0,0 +1,30 @@
+import { getBearerToken } from "../utils/auth.js";
+import { validateGeneratePostBody } from "../models/postModels.js";
+import { generatePostService } from "../services/postService.js";
+
+export async function generatePost(req, res, next) {
+	try {
+		const token = getBearerToken(req);
+
+		const body = validateGeneratePostBody({
+			repo: req.body?.repo,
+			commits: req.body?.commits, // optional
```

Comment:
```suggestion
			commits: req.body?.commits,
```

옵셔널 체이닝이 있어서 주석은 없어도 되겠습니다~